thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1829,16 +1863,16 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
         }
         virtual void getSubFileStats(std::vector<OwnedPtr<CRuntimeStatisticCollection>> & subFileStats) override
         {
-            ISuperFileDescriptor * superFDesc = allParts->item(0).queryOwner().querySuperFileDescriptor();
             for (size_t i=0; i<parts.size(); i++)
             {
-                unsigned partNo = parts[i] & partMask;
-                unsigned subfile, lnum;
                 PartIO partIO = partIOs[i];
```

Comment:
this should move inside the conditon below,
Should also change so it refernces the PartIO object, it's currently copying it (which involves linking all it's members etc).